transfers/transfer.py

```diff
@@ -189,8 +200,11 @@ def get_next_transfer(ss_url, ts_location_uuid, path_prefix, depth, completed):
     browse_info = _call_url_json(url, params)
     if browse_info is None:
         return None
-    dirs = browse_info['directories']
-    dirs = map(base64.b64decode, dirs)
+    if see_files:
+        dirs = browse_info['entries']
```

Comment:
Extreme nitpick, but since this is no longer always directories, maybe worth renaming the variable?